compose/ui/ui/src/webTest/kotlin/androidx/compose/ui/input/TextInputTests.kt

```diff
@@ -78,19 +73,28 @@ class TextInputTests : OnCanvasTests {
         }
     }
 
+    private fun WebApplicationScope.assertTextFieldValue(expected: String, actual: MutableState<TextFieldValue>, message: String? = null) {
+        assertEquals(expected = expected, actual = actual.value.text, message = message)
+    }
+
+    private fun WebApplicationScope.assertTextFieldValue(expected: String, actual: InputInteractor, message: String? = null) {
+        assertEquals(expected = expected, actual.textFieldValues[0].value.text, message = message)
+    }
+
     private suspend fun InputInteractor.createTextFieldWithChannel(
         content: @Composable () -> Unit = {
             val focusRequester = focusRequesters[0]
 
             val textState = remember {
-                mutableStateOf("")
+                mutableStateOf(TextFieldValue())
             }
 
+            this@createTextFieldWithChannel.textFieldValues.add(textState)
```

Comment:
Hmm, you're adding a `MutableState<TextFieldValue>` to the list on every recomposition. Why not just create it once in the `InputInteractor`, and only access it here?